README.md

```diff
@@ -1,6 +1,6 @@
 # crunchy-data-postgresql-stig-baseline
 
-InSpec profile to validate the secure configuration of Crunchy Data PostgreSQL against [DISA's](https://public.cyber.mil/stigs/downloads/) Crunchy Data PostgreSQL Security Technical Implementation Guide (STIG) Version 2, Release 1. (Applies to database versions 10, 11, 12 & 13)
+InSpec profile to validate the secure configuration of Crunchy Data PostgreSQL against [DISA's](https://public.cyber.mil/stigs/downloads/) Crunchy Data PostgreSQL Security Technical Implementation Guide (STIG) Version 2, Release 2. (Applies to database versions 10, 11, 12 & 13)

```

Comment:
We should also add the new input to the inputs section of the README

Reply:
How do you mean? The inputs_postgres12.yml?

Reply:
I mean in the instructions of the README here
https://github.com/mitre/crunchy-data-postgresql-stig-baseline/tree/3eb6257a2fb3990e91d51f022876822a1317c35d?tab=readme-ov-file#example-inputs-you-can-use